src/Iris/Env.hs

```diff
@@ -1,4 +1,8 @@
 {-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ApplicativeDo #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
+{-# HLINT ignore "Use >=>" #-}
```

Comment:
Where does `hlint` want to use this? 🤔

Reply:
In the `for_` block here:

```
for_ cliEnvSettingsRequiredTools $ \tool ->
        checkTool cmd tool >>= \case
            ToolOk  -> pure ()
            toolErr -> throwIO $ CliEnvException $ CliEnvToolError toolErr
```

I thought the use of `>=>` might be confusing to read. Should I follow hlint's suggestion instead?

Reply:
I see. No need to use `>=>`, it's indeed less clear as you suggested. I haven't configured HLint for this project so I'll take care of all hlint warnings once it's set up 🙂 

For now, you can remove flags responsible for disabling HLint warnings. I imagine, a configuration in the future will control everything from a config file.

Reply:
Done!